code/__DEFINES/citadel_defines.dm

```diff
@@ -12,6 +12,7 @@
 #define ui_boxcraft "EAST-4:22,SOUTH+1:6"
 #define ui_boxarea "EAST-4:6,SOUTH+1:6"
 #define ui_boxlang "EAST-5:22,SOUTH+1:6"
+#define ui_boxvore	"EAST-4:22,SOUTH+1:6"
```

Comment:
wouldnt this value make the button overlap with the crafting button when the resolution is set to 1:1? or am i dumb and missing something?

Reply:
No because the button is flipped


![](https://i.gyazo.com/156680de6bd104ea42d2a5e34b506b31.png)